woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-user-edit-handler.php

```diff
@@ -56,6 +56,8 @@ class SV_WC_Payment_Gateway_Admin_User_Edit_Handler {
 	/**
 	 * Initialize and setup the Admin User Edit Handler
 	 *
+	 * TODO: $text_domain is deprecated, remove argument on next breaking release
+	 *
 	 * @since 3.0.0
 	 */
 	public function __construct( $plugin, $text_domain ) {
```

Comment:
apologies, after thinking about what we're doing for the 4.1 milestone, we can break backwards compat for gateways but not for all plugins, so in this specific intstance it's fine to pull the `$text_domain` param out :)